embrace-android-sdk/api/embrace-android-sdk.api

```diff
@@ -375,12 +375,10 @@ public final class io/embrace/android/embracesdk/spans/EmbraceSpanEventJsonAdapt
 	public fun toString ()Ljava/lang/String;
 }
 
-public final class io/embrace/android/embracesdk/spans/ErrorCode : java/lang/Enum, io/embrace/android/embracesdk/internal/spans/EmbraceAttributes$Attribute {
+public final class io/embrace/android/embracesdk/spans/ErrorCode : java/lang/Enum {
```

Comment:
This is marked as @BetaApi so I think this change is OK, especially given that customers shouldn't be using those methods anyway.